### ISSUE TITLE
Unable to configure the increment value for new releases

### ISSUE THREAD

**Issue by ssbarnea** [release-drafter/release-drafter#1397]
Some systems have very strict constraints regarding on what version numbers are valid and what can be used for pre-releases or not. Such an example is https://marketplace.visualstudio.com/ which is using following https://code.visualstudio.com/api/working-with-extensions/publishing-extension#prerelease-extensions 

> So, we recommend that extensions use major.EVEN_NUMBER.patch for release versions and major.ODD_NUMBER.patch for pre-release versions. For example: 0.2.* for release and 0.3.* for pre-release.

They recommend using even and odd to distinguish between releases and prereleases, but that cannot really be configured in dependabot because the increment values are hardcoded to 1. If we could change the value we could probably make it work.

There are other problems caused by the very strict versioning requirements they have related to continuous deployment of pre-releases, but even these could have probably sorted with a custom increment value for the patch.

- Be able to setup a custom value for default PATCH value instead of `0`. Setting it to something like `1000` could allow CI/CD to keep building each additional merge to branch and generate a number for it that would be smaller than the next release number.
- Be able to configure a `{MAJOR|MINOR|PATCH}_INCREMENT` like `1000` instead of current hardcoded 1.

If these are configurable, projects publishing to vscode marketplace could make use of them for versioning.

**Comment by cchanche** [comment]
Yeah so what VSCode suggests people do with their version is quite intrusive IMO. You loose quite a lot of control over your release versions, and have to get out of any automatic increments (semantics or not).

What about adding config (or input) like :

```yaml
patch-increment: number # default 1
minor-increment: number # default 1
major-increment: number # default 1
```

If I get thir right, one would be able to use this in conjunction with `$NEXT_PATCH_VERSION` `$NEXT_MINOR_VERSION` and `$NEXT_MAJOR_VERSION`.

Ex: 

```
# .github/release-drafter.prerelease.yml
include-prereleases: true
prerelease: true
prerelese-identifier: ''
minor-increment: 2
name-template: 'Prerelease v$NEXT_MINOR_VERSION'
tag-template: 'v$NEXT_MINOR_VERSION'
```
```
# .github/release-drafter.release.yml
include-prereleases: false
prerelease: false
minor-increment: 2
name-template: 'Release v$NEXT_MINOR_VERSION'
tag-template: 'v$NEXT_MINOR_VERSION'
```
